Makefile

```diff
@@ -95,6 +95,28 @@ golangci-lint:
 test:
 	$(GO) test -v -mod=readonly -failfast -race ./...
 
+MATCH=v.+\/mattermost-load-test-ng-v.+-linux-amd64.tar.gz
+REPLACE=$(NEXT_VER)\/mattermost-load-test-ng-$(NEXT_VER)-linux-amd64.tar.gz
+TAG_EXISTS=$(shell git rev-parse $(NEXT_VER) >/dev/null 2>&1; echo $$?)
+
+release:
+	@if [[ -z "${NEXT_VER}" ]]; then \
+		echo "Error: NEXT_VER must be defined"; \
+		exit -1; \
+	else \
+		if [[ "${TAG_EXISTS}" -eq 0 ]]; then \
+		  echo "Error: tag ${NEXT_VER} already exists"; \
+			exit -1; \
+		else \
+			for file in $(shell grep -rPl --include="*.go" --include="*.json" $(MATCH)); do \
+			sed -r -i 's/$(MATCH)/$(REPLACE)/g' $$file; \
+			done; \
+			git commit -a -m 'Releasing $(NEXT_VER)'; \
+			git tag $(NEXT_VER); \
+			goreleaser --rm-dist; \
```

Comment:
Please also add a check to confirm that goreleaser is installed, with a link to the install page if the command is not found.

Reply:
Sounds good :+1:

Reply:
😎